### PR TITLE
Contact card refactor

### PR DESCRIPTION
**Pull request by ColinStClaire**

### CODE REVIEW

File: TicketDesk/ngWebClient-StandAlone/src/app/contact-info/contact-info.component.ts [hunk 1]
```diff
@@ -2,23 +2,28 @@ import { AdContactService } from './../services/ad-contact.service';
...
 
   ngOnInit() {
-    this.contactService.getContactCardInfo(this.ownerId)
-      .subscribe(res => {
-        this.user = res;
+    this.populateContactCard();
+  }
+
+  private populateContactCard() {
+    const { contactService, userName } = this;
```
Comment (kixiQu) [inline TicketDesk/ngWebClient-StandAlone/src/app/contact-info/contact-info.component.ts]: the tall poppy must be mowed down

File: TicketDesk/ngWebClient-StandAlone/src/app/models/ticket-stub.ts [hunk 2]
```diff
@@ -12,5 +12,14 @@ export interface TicketStub {
   }
 
 // Headings for table (displayed to User)
-export const columnHeadings: string[] = ['TicketID', 'Title', 'Status', 'Priority', 'Owner', 'Assigned To', 'Category',
-                                      'Created Date', 'Last Updated'];
+export const columnHeadings: string[] = [
+  'TicketID',
```
Comment (kixiQu) [inline TicketDesk/ngWebClient-StandAlone/src/app/models/ticket-stub.ts]: my guess would be that this should be Ticket ID

File: TicketDesk/ngWebClient-StandAlone/src/app/services/single-ticket.service.ts [hunk 3]
```diff
@@ -23,15 +23,20 @@ export class SingleTicketService {
...
+    const url = settings.ticketDetailsURL + ticketId.toString();
+    console.warn(`getting details for ticketId: ${ticketId} from url: ${url}`);
+    return this.http.get(url)
       .pipe(catchError(this.handleError));
   }
 
-  getOwner(ticketId: number) {
-  // todo: refactor this into the ticket itself
-  }
+  // TODO: implement to get assests such as links
```
Comment (kixiQu) [inline TicketDesk/ngWebClient-StandAlone/src/app/services/single-ticket.service.ts]: s/assests/assets/

File: TicketDesk/ngWebClient-StandAlone/src/app/single-ticket-view/single-ticket-view.component.html [hunk 4]
```diff
@@ -12,7 +12,7 @@ <h6 *ngIf="ticketIsOpen(); else closedTicket" class="card-title">Active
               <span class="fa fa-envelope" 
               aria-hidden="true"></span>
           </ng-template>
-          <h6 class="card-title">
+          <h6 class="card-title">Ticket Id:
```
Comment (kixiQu) [inline TicketDesk/ngWebClient-StandAlone/src/app/single-ticket-view/single-ticket-view.component.html]: s/Id/ID/

File: TicketDesk/ngWebClient-StandAlone/src/app/app-settings.ts [hunk 5]
```diff
@@ -1,4 +1,5 @@
 export const serviceBaseURL = 'http://localhost:50067/api';
+export const adUserURL = serviceBaseURL + '/users/';
```
Comment (kixiQu) [inline TicketDesk/ngWebClient-StandAlone/src/app/app-settings.ts]: we have space here for this to be `activeDirectoryUserURL`

Comment (kixiQu) [reply TicketDesk/ngWebClient-StandAlone/src/app/app-settings.ts]: And I think that would make it usefully more clear

Comment (kixiQu) [reply TicketDesk/ngWebClient-StandAlone/src/app/app-settings.ts]: but I'm just commenting to have you consider the idea, not a change necessary for approval

**Review by kixiQu — CHANGES_REQUESTED** [review]
molto bene

File: TicketDesk/ngWebClient-StandAlone/src/app/contact-info/contact-info.component.ts (hunk 1 above)
Comment (ColinStClaire) [reply TicketDesk/ngWebClient-StandAlone/src/app/contact-info/contact-info.component.ts]: fixed

File: TicketDesk/ngWebClient-StandAlone/src/app/models/ticket-stub.ts (hunk 2 above)
Comment (ColinStClaire) [reply TicketDesk/ngWebClient-StandAlone/src/app/models/ticket-stub.ts]: fixed

File: TicketDesk/ngWebClient-StandAlone/src/app/services/single-ticket.service.ts (hunk 3 above)
Comment (ColinStClaire) [reply TicketDesk/ngWebClient-StandAlone/src/app/services/single-ticket.service.ts]: fixed

File: TicketDesk/ngWebClient-StandAlone/src/app/single-ticket-view/single-ticket-view.component.html (hunk 4 above)
Comment (ColinStClaire) [reply TicketDesk/ngWebClient-StandAlone/src/app/single-ticket-view/single-ticket-view.component.html]: fixed

**Review by kixiQu — DISMISSED** [review]
it so bootiful

**Review by kixiQu — APPROVED** [review]
Can I approve my own merge?

**Pull request merged**